ExtraHop/revealx_360/tests/test_1.json

```diff
@@ -0,0 +1,45 @@
+{
+  "input": {
+    "message": "{\"id\": 22222222222, \"start_time\": 1701379823296, \"update_time\": 1706720536009, \"mod_time\": 1706720577690, \"title\": \"Deprecated SSL/TLS Versions\", \"description\": \"db1\\\\.example\\\\.org established an SSL/TLS connection with a deprecated version of SSL/TLS. SSL 2.0, SSL 3.0, and TLS 1.0 are deprecated because they are vulnerable to attacks.\", \"risk_score\": 30, \"type\": \"deprecated_ssl_tls_individual\", \"recommended_factors\": [], \"recommended\": false, \"categories\": [\"sec\", \"sec.hardening\"], \"properties\": {\"version\": \"TLSv1.0\"}, \"participants\": [{\"role\": \"offender\", \"object_id\": 33333333333, \"object_type\": \"device\", \"object_value\": \"1.2.3.5\", \"hostname\": \"db1.example.org\", \"id\": 2222, \"external\": false, \"scanner_service\": null}], \"ticket_id\": null, \"assignee\": null, \"status\": null, \"resolution\": null, \"mitre_tactics\": [], \"mitre_techniques\": [], \"appliance_id\": 3, \"is_user_created\": false}",
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Reveal(x) 360",
+        "dialect_uuid": "1df44c62-33d3-41d4-8176-f1fa13589eea"
+      }
+    }
+  },
+  "expected": {
+    "message": "{\"id\": 22222222222, \"start_time\": 1701379823296, \"update_time\": 1706720536009, \"mod_time\": 1706720577690, \"title\": \"Deprecated SSL/TLS Versions\", \"description\": \"db1\\\\.example\\\\.org established an SSL/TLS connection with a deprecated version of SSL/TLS. SSL 2.0, SSL 3.0, and TLS 1.0 are deprecated because they are vulnerable to attacks.\", \"risk_score\": 30, \"type\": \"deprecated_ssl_tls_individual\", \"recommended_factors\": [], \"recommended\": false, \"categories\": [\"sec\", \"sec.hardening\"], \"properties\": {\"version\": \"TLSv1.0\"}, \"participants\": [{\"role\": \"offender\", \"object_id\": 33333333333, \"object_type\": \"device\", \"object_value\": \"1.2.3.5\", \"hostname\": \"db1.example.org\", \"id\": 2222, \"external\": false, \"scanner_service\": null}], \"ticket_id\": null, \"assignee\": null, \"status\": null, \"resolution\": null, \"mitre_tactics\": [], \"mitre_techniques\": [], \"appliance_id\": 3, \"is_user_created\": false}",
+    "event": {
+      "category": ["intrusion_detection"],
+      "code": "deprecated_ssl_tls_individual",
+      "kind": "alert",
+      "reason": "db1\\.example\\.org established an SSL/TLS connection with a deprecated version of SSL/TLS. SSL 2.0, SSL 3.0, and TLS 1.0 are deprecated because they are vulnerable to attacks.",
+      "start": "2023-11-30T21:30:23.296000Z",
+      "type": ["info"]
+    },
+    "@timestamp": "2024-01-31T17:02:57.690000Z",
+    "extrahop": {
+      "revealx360": {
+        "categories": ["sec", "sec.hardening"],
+        "id": "22222222222",
+        "title": "Deprecated SSL/TLS Versions"
+      }
+    },
+    "host": {
+      "id": "33333333333",
+      "ip": "1.2.3.5",
```

Comment:
This field must be an array (cf [specification](https://www.elastic.co/guide/en/ecs/current/ecs-host.html#field-host-ip))
```suggestion
      "ip": ["1.2.3.5"],
```